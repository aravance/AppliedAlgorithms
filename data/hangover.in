1.00
3.71
0.04
5.19
5.20
0.01
0.60
0.90
1.20
1.40
1.50
1.60
1.80
1.90
2.00
0.00
