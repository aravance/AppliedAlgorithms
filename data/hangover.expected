3 card(s)
61 card(s)
1 card(s)
273 card(s)
276 card(s)
1 card(s)
2 card(s)
3 card(s)
4 card(s)
5 card(s)
6 card(s)
7 card(s)
8 card(s)
9 card(s)
10 card(s)
